JS.md

```diff
@@ -199,14 +199,16 @@ Array exports(moduleObject)
 If `moduleObject` is not a `WebAssembly.Module`, a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror)
 is thrown.
 
-This function returns an `Array` produced by mapping each
+This function returns a new `Array` every time it is called. Each such `Array` is produced by mapping each
 [`Ast.export`](https://github.com/WebAssembly/spec/blob/master/interpreter/spec/ast.ml#L152)
 `e` of [moduleObject.[[Module]].exports](https://github.com/WebAssembly/spec/blob/master/interpreter/spec/ast.ml#L187)
 to the Object `{ name: String(e.name), kind: e.ekind }` where `e.name` is [decoded as UTF8](Web.md#names)
 and `e.ekind` is mapped to one of the String values `"function"`, `"table"`, `"memory"`, `"global"`.
 
 Note: other fields like `signature` may be added in the future.
 
+The returned `Array` is populated in the same order as that in which exports appeared in the WebAssembly binary's exports table.
```

Comment:
Nit: simplify sentence to "... is populated in the same order in which exports appear ..." (Also note present tense.)